types/react-onsenui/index.d.ts

```diff
@@ -392,6 +392,8 @@ export class Card extends Component<{
     modifier?: string,
 }, any> {}
 
+/** Controls */
```

Comment:
This line is probably a merge failure. I guess it should come above the new added components.

Reply:
This is here on purpose to mark that the components below are from the Controls section in the docs.